### PR TITLE
docs: tokens page titles

### PR DESCRIPTION
**Pull request by bennypowers**
## Things to fix

- [x] Truncate the titles on the tokens pages
- [x] Add flex gap between search and heading @nikkimk 
- [x] Add padding bottom on header bar @nikkimk 
- [x] Font-size of placeholder text in searchon tokens pages is larger than on the Overview page @nikkimk 
- [x] Remove gap on mobile @nikkimk 
- [x] Remove scrolling by stacking search under heading with flex-orientation: column @nikkimk 

### 1, 2, 3:
Screenshot 2023-07-05 at 10 19 50 AM

### 4:
image

### 5, 6:
Screenshot 2023-07-05 at 10 28 05 AM

## Notes to Reviewers
feel free to add as many design tweaks to the tokens pages as you want before merging this, or not

### CODE REVIEW

File: docs/tokens/tokens.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 templateEngineOverride: njk
 permalink: /tokens/{{ cat.slug }}/index.html
 eleventyComputed:
-  title: '{{ (cat.title or cat.slug) | capitalize }} | Tokens | Red Hat Design System'
+  title: '{{ (cat.title or cat.slug) | capitalize }}'
```
Comment (bennypowers) [inline docs/tokens/tokens.html:6]:
```suggestion
  title: '{{ (cat.title or cat.slug) | capitalize }} Tokens'
```

File: docs/tokens/tokens.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 templateEngineOverride: njk
 permalink: /tokens/{{ cat.slug }}/index.html
 eleventyComputed:
-  title: '{{ (cat.title or cat.slug) | capitalize }} | Tokens | Red Hat Design System'
+  title: '{{ (cat.title or cat.slug) | capitalize }} Tokens'
```
[the remark this answers is not in the document]
Comment (bennypowers) [reply docs/tokens/tokens.html:6]: :shrug:  https://www.grammarly.com/blog/capitalization-in-the-titles/

[the remark this answers is not in the document]
Comment (bennypowers) [reply docs/tokens/tokens.html:6]: I leave that decision to your good graces :smiley:

[the remark this answers is not in the document]
Comment (nikkimk) [reply docs/tokens/tokens.html:6]: @bennypowers I'll make the change.

For reference, I'm going by https://www.redhat.com/en/about/brand/standards/typography:

> Use sentence case
> We use sentence case—not all caps or title case—in Red Hat materials, including headlines (we use title case for job titles). Type in all caps implies that we are yelling or overly aggressive. Sentence case is more authentic, friendly, and realistic, and it lets capitalized words stand out. For guidance on capitalizing words, review the [Red Hat corporate style guide](https://source.redhat.com/groups/public/word-nerds/red_hat_word_nerds_wiki/corporate_style_guide).

**Review by nikkimk — APPROVED** [review]
Lone ghost tries mischief

**Pull request merged**